docs/modules/session-explorer/pages/analytics/review-network-payload-data.adoc

```diff
@@ -16,13 +16,13 @@ Next, review xref:_review_payload_data[payload data] or xref:_review_response_ti
 [#_review_payload_data]
 == Review payload data
 
```

Comment:
Do we need something to say that Network Payload Capture must be set up to have data here. And point to NPC docs? https://docs.kobiton.com/devices/local-devices/network-payload-capture/

Reply:
Should we point to [Create a configuration](https://docs.kobiton.com/devices/local-devices/network-payload-capture/create-a-configuration)? Something like: Ensure that you [create a configuration](https://docs.kobiton.com/devices/local-devices/network-payload-capture/create-a-configuration) to enable network payload capture.